src/lib/airtable/request.js

```diff
@@ -508,18 +509,22 @@ export const updateCustomer = async (id, recordUpdates) => {
 
 // NONGENERATED: Edit customer
 export const editCustomer = async (customer) => {
-  try {
-    const resp = await fetch(`${process.env.REACT_APP_AIRTABLE_ENDPOINT_URL}/customers/edit`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(customer)
-    })
-
-    console.log(resp);
-  } catch (err) {
-    console.log(err);
+  if (!customer.id) {
+    addToOfflineCustomer(customer, 'edits', customer.customerUpdate);
```

Comment:
We also have to editCustomerInRedux after the if statement to update the redux store (maybe this just happens after the `if / else`?